tests/test_branches_utils.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+
+from trie.trie import (
+    BinaryTrie,
+)
+from trie.utils.branches import (
+    if_branch_exist,
+)
+
+
+def set_up_test_trie():
```

Comment:
*small refactor suggestion*

If you made this into a fixture:

```python
@pytest.fixture
def trie(...):
    ...
```

Then you could just include it in the arguments of `test_branch_exist` and use it directly as it will be reset for each test.